packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -98,43 +95,20 @@ const Updates = decorate([
         const { effects, state } = this
         await xoaUpdater.configure({
           ...pick(state, [
+            '_channel',
```

Comment:
Unfortunately this does not work because you are send `_channel` to the updater and not `channel`.

Reply:
Maybe rename as `channel` and rename the computed (e.g. `consolidatedChannel)?